src/functionalTest/java/uk/gov/hmcts/reform/laubackend/idam/serenityfunctionaltests/runner/DeletedAccountsApiTest.java

```diff
@@ -80,5 +88,95 @@ public void assertHttpForbiddenWithInvalidS2SToken() throws JsonProcessingExcept
         );
     }
 
+    @Test
+    @Title("Assert response code of 201 for GET Request deletedAccounts")
+    public void assertHttpSuccessResponseCodeForGetRequestCaseViewApi()
+        throws JsonProcessingException, JSONException {
+
+        String serviceToken = getApiSteps.givenAValidServiceTokenIsGenerated(
+            TestConstants.USER_DISPOSER_SERVICE_NAME);
+
+        String authToken = getApiSteps.validAuthorizationTokenIsGenerated();
+        Map<String, String> queryParams = getApiSteps.givenValidParamsAreSuppliedForGetLogonApi();
 
+        Response response = getApiSteps.whenTheGetServiceIsInvoked(
+            TestConstants.DELETED_ACCOUNTS_ENDPOINT,
+            serviceToken,
+            authToken,
+            queryParams
+        );
+
+
+        String successOrFailure = getApiSteps.thenASuccessResposeIsReturned(response);
+        Assert.assertEquals(
+            successOrFailure,
+            TestConstants.SUCCESS,
+            "DeletedAccounts POST API response code 201 assertion is not successful"
+        );
+    }
+
+    @Test
+    @Title("Assert response code bad request without start timestamp")
+    public void assertHttpBadRequestWithoutStartTimestamp() throws JsonProcessingException {
+
+        String serviceToken = getApiSteps.givenAValidServiceTokenIsGenerated(
+            TestConstants.USER_DISPOSER_SERVICE_NAME);
+        DeletedAccountsRequest request = getApiSteps.generateInvalidDeletedAccountsSearchRequest();
+
+        Response response = getApiSteps.whenThePostServiceIsInvoked(
+            TestConstants.DELETED_ACCOUNTS_ENDPOINT,
+            serviceToken,
+            request
+        );
+        String successOrFailure = getApiSteps.thenBadResponseIsReturned(response, 400);
+        Assert.assertEquals(
+            successOrFailure,
+            TestConstants.SUCCESS,
+            "DeletedAccountsSearch GET API response code 400 assertion is not successful"
+        );
+    }
+
+    @Test
+    @Title("Assert response code bad request without start timestamp")
+    public void assertUnauthorizedRequest() throws JsonProcessingException, JSONException {
+
+        String serviceToken = getApiSteps.givenAValidServiceTokenIsGenerated(
+            TestConstants.USER_DISPOSER_SERVICE_NAME);
+
+        Map<String, String> header = new HashMap<>();
+
+        Response response = getApiSteps.whenTheGetServiceIsInvoked(
```

Comment:
You could use `performGetOperation(...)` directly here, then you wouldn't need to create additional method in BaseSteps.java

Reply:
Done